pypif_sdk/func/Chem_formula_split.py

```diff
@@ -0,0 +1,55 @@
+import re
+bad_chem_form = "CH3.4[[CH2]$_{40}$(CH)4]2[CH]10CH3"
+#double check decimals work
+def expand_formula(formula_string):
```

Comment:
Was this an older version of the functionality?  It looks mostly duplicated by `calculate_funcs.py`

Reply:
Yeah.... That while file should be gone